crates/ruff_python_ast/src/context.rs

```diff
@@ -203,6 +206,75 @@ impl<'a> Context<'a> {
         }
     }
 
+    /// Given a `module` and `member`, return the fully-qualified name of the binding in the current
+    /// scope, if it exists.
+    ///
+    /// E.g., given:
+    ///
+    /// ```python
+    /// from sys import version_info as python_version
+    /// print(python_version)
+    /// ```
+    ///
+    /// ...then `resolve_binding("sys", "version_info")` will return `Some("python_version")`.
+    pub fn resolve_binding(&self, module: &str, member: &str) -> Option<String> {
```

Comment:
Nit: You could avoid allocating in some cases by returning `Option<Cow<str>>`, but I don't know if its worth here or if it would be better to return an `enum` instead that then has a method to get the full qualified name of the local symbol referencing the import.